cmd/sink/main.go

```diff
@@ -124,7 +124,7 @@ func (sink *Sink) Receive(ctx context.Context, event cloudevents.Event) {
 
 func main() {
 	logger := log.New(os.Stderr, "", log.LstdFlags|log.Lmicroseconds|log.LUTC)
-	err := kaObservability.Start()
+	err := kaObservability.Start("kubearchive.sink")
```

Comment:
I think `"kubearchive.sink"` should be a constant instead of a magic string

Reply:
Moved to the top as a constant.